Fundamentals/Backoffice/Property-Editors/Built-in-Property-Editors/Block-Grid-Editor/index.md

```diff
@@ -0,0 +1,625 @@
+---
+versionFrom: 9.0.0
+versionTo: 10.0.0
+---
+
+# Block Grid
+
+`Alias: Umbraco.BlockGrid`
+
+`Returns: BlockGridModel`
+
+The **Block Grid** enables the editor to layout their content in the Umbraco backoffice. The content is made of blocks, which can contain simple or complex data. By defining how many rows and columns the block spans, each Block gets a defined size. This will make them appear next to each other, together, or even in formations.
+
+Additionally, Blocks can nest other Blocks forming more complex or strict compositions.
+
+The default provided layout mechanism is based on CSS Grids. This can be adjusted or replaced to achieve the right layout for your project.
+
+## Configure the Block Grid
+
+The Block Grid property editor is configured in the same way standard property editors, which is via the Data Types backoffice interface.
+
+To set up your Block Grid Editor property, follow these steps:
+
+1. Navigate to the **Settings** section in the Umbraco backoffice.
+2. Locate and right-click the **Data Types** folder.
+3. **Create a new Data Type**.
+4. Select **Block Grid** from the list of available property editors.
+
+You will see the configuration options for a Block Grid as shown below.
+
+![Block Grid - Data Type Definition](images/BlockListEditor_DataType.jpg)
+> TODO: screenshot
+
+The Data Type editor allows you to configure the following properties:
+
+- **Available Blocks** - Define the Block Types to be available for use in the property. Read more on how to set up Block Types below.
+- **Amount** - Sets the minimum and/or maximum number of blocks that should be allowed in the root of the layout.
+- **Live editing mode** - Enabling this will enable you to see the changes you making as you are making them.
+- **Property editor width** - Overwrite the width of the property editor. This field takes any valid CSS value for "max-width".
+- **Grid Columns** - Define the number of columns in your grid layout. Default is 12 columns.
+- **Layout Stylesheet** - Replace the built-in Layout Stylesheet. Additionally you can retrieve the default layout stylesheet to use as base for your own or inspiration for writhing your own.
+
+## Setup Block Types
+
+Block Types are based on **Element Types**. These can be created beforehand or while setting up your Block Types.
```

Comment:
Do we need here to explain how to create an Element Type as a Block Type?

Reply:
We didn't do that in List, so I would say not needed.
But when that said it would be a nicer way for newcomers. So a link to an article on that would be great!

Reply:
Noted 💪 
We have not documented Element Types yet, but I've added a task to our backlog.